sqlmesh/core/config/connection.py

```diff
@@ -407,6 +408,7 @@ class SnowflakeConnectionConfig(ConnectionConfig):
     role: t.Optional[str] = None
     authenticator: t.Optional[str] = None
     token: t.Optional[str] = None
+    application: str = 'Tobiko_SQLMesh'
```

Comment:
this shouldn’t be configurable

Reply:
made it a property

Reply:
Nevermind, made it a classvar instead. Let me know if you want more protection on it.